### DISCUSSION TITLE
Has C#10 supported fixed-size buffer?

### DISCUSSION THREAD

**Discussion by ygc369** [dotnet/csharplang#5264] [Q&A]
Has C#10 supported fixed-size buffer? If not yet, will it be supported in C#11?

**Comment by jnm2** [comment]
Are you referring to https://github.com/dotnet/csharplang/issues/1314 specifically or the [fixed-size buffer feature](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/unsafe-code#fixed-size-buffers) that has been around since (I think) C# 1.0?

**Comment by ygc369** [comment]
@jnm2 
I'm referring to #1314.

**Reply by CyrusNajmabadi** [reply]
@ygc369  you can see the status in your link:

![image](https://user-images.githubusercontent.com/4564579/136682837-9e9272f2-a314-499a-9438-38a62f19034c.png)

**Reply by huoyaoyuan** [reply]
@CyrusNajmabadi I'm curious about how do you pick features to discuss in LDM. Do you have a priority list? Do you sort issues by votes? Do you have a list about implicit feature dependencies?

**Reply by CyrusNajmabadi** [reply]
Ldm members decide what they think is important and bring their ideas/issues/discussion to the meetings.

**Comment by ygc369** [comment]
I remember that this champion was ever put into C#10 feature list. Why is it removed from C#10 finally?

**Reply by CyrusNajmabadi** [reply]
Because we didn't get around to it in c#10.

**Reply by ygc369** [reply]
So has it been put into C#11 feature lists?
